openrr-teleop/src/ik.rs

```diff
@@ -231,3 +231,37 @@ fn default_move_step_linear() -> [f64; 3] {
 fn default_step_duration_secs() -> f64 {
     0.1
 }
+
+#[cfg(test)]
+mod tests {
+    use assert_approx_eq::*;
+
+    use super::*;
+
+    #[test]
+    fn test_default_move_step_angular() {
+        let def = default_move_step_angular();
+
+        assert_approx_eq!(def.get(0).unwrap(), 0.05_f64);
+        assert_approx_eq!(def.get(1).unwrap(), 0.05_f64);
+        assert_approx_eq!(def.get(2).unwrap(), 0.17_f64);
+        assert_eq!(def.get(3), None);
```

Comment:
I prefer to use index access(i.e. def[0], def[1], def[2]), and to confirm length of the array.
But it's OK if it works.

Reply:
I think using `.get()` everytime is bad choice.
However, using it is good if length is not obvious.

In the test, I want to check including length.

Reply:
P.S.
This code is working and `def.get(*)` is returned `Some(def[*])` if elements exist.